healthcareai/common/filters.py

```diff
@@ -1,6 +1,25 @@
 import pandas as pd
+from sklearn.preprocessing import StandardScaler
 
 def remove_datetime_columns(df):
     # TODO: make this work with col names shorter than three letters
     cols = [c for c in df.columns if c[-3:] != 'DTS']
-    return df[cols]
\ No newline at end of file
+    return df[cols]
+
+def feature_scaling(X_train,X_test,columns_to_scale):
```

Comment:
Thanks, Coco! Would you mind adding in a doc string with the explanations as to arguments, functionality, etc?